utils/pipeline/pipeline_test.go

```diff
@@ -40,8 +40,9 @@ func TestPipeline(t *testing.T) {
 			assert.False(t, open)
 		})
 		t.Run("Bridge", func(t *testing.T) {
-			_, open := <-Bridge[int](ctx, nil)
-			assert.False(t, open)
+			out := make(chan int)
+			chOfInts := make(chan (<-chan int))
+			go Bridge[int](ctx, out, chOfInts) // non-blocking
```

Comment:
What's the purpose of this test if we don't assert anything here?

Reply:
If we want to assert that the function is running and waiting for input, we can actually do something like this:
```suggestion
			go func() {
						Bridge[int](ctx, out, chOfInts)
						select {
						case t.Context().Done():
						default:
									assert.FailNow(...)
						}
			}()
```

Reply:
Ah yeah, the Bridge function probably shouldn't run in a separate routine. I've updated the test so it doesn't run in a separate routine.